docs/getting-started/mobile-ui-integration.md

```diff
@@ -0,0 +1,75 @@
+---
+id: mobile-required-ui
+title: Required UI for Mobile Integration
+slug: /getting-started-mobile-required-ui
+---
+
+# User Interface requirements
+
+Currently Nimbus provides no user-interface components of its own, though provides API to connect to existing settings screens.
+
+## Global opt-out/opt-in for experiments
+
+The settings page should include a `Studies` toggle, which allows users to opt-in or opt-out of experiments. The example from Firefox for iOS is shown:
+
+<img src="/img/firefox-ios/studies-toggle.png" width="300px" />
+
+Toggling `Studies` flag on should `globalUserParticipation`:
```

Comment:
This wording seems off – I'm guessing this wording was maybe adjusted a few times and parts of both versions are present. I don't know if that's the case here I just know it's something I've done before 😂 

Maybe something like 
```suggestion
Toggling the `Studies` flag should set the `Nimbus` value for `globalUserParticipation`:
```